src/Model/PhasorDynamics/Governor/Tgov1/Tgov1Impl.hpp

```diff
@@ -286,10 +294,11 @@ namespace GridKit
         // Input Variables
         if (signals_.template isAttached<Tgov1ExternalVariables::DELTAOMEGA>())
         {
-          omega_ = signals_.template readExternalVariable<Tgov1ExternalVariables::DELTAOMEGA>();
+          we_[0]         = signals_.template readExternalVariable<Tgov1ExternalVariables::DELTAOMEGA>();
+          we_indices_[0] = signals_.template readExternalVariableIndex<Tgov1ExternalVariables::DELTAOMEGA>();
         }
 
-        evaluateInternalResidual(y_.data(), yp_.data(), w_.data(), f_.data());
+        evaluateInternalResidualWithExternal(y_.data(), yp_.data(), wb_.data(), we_.data(), f_.data());
```

Comment:
Why this name change? The previous name `evaluateInternalResidual` was clear. Internal residual evaluation should include external (coupling) variables.